### PR TITLE
kb-exchangerates

### PR DESCRIPTION
**Pull request by christianbrown01**

### CODE REVIEW

File: kb-exchangerates [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# International Payment, Currency and Exchange
```
Comment (chrislittle) [inline kb-exchangerates]: I don't see any of the metadata required on any kb article.
refer to https://github.com/CenturyLinkCloud/PublicKB/blob/master/README.md

Comment (chrislittle) [inline kb-exchangerates]: There also seems to be some issue with the kb category.  I don't see the kb folder this is being added to.  you must put the kb in a category I believe

**Comment by rpdubose** [comment]
@christianbrown01 Is this true that a WEBC customer can change their desired currency?

**Comment by srveit** [comment]
Closing because of age. Feel free to resubmit after comments have been addressed.

**Pull request closed without merging**